src/Microsoft.AspNetCore.StaticFiles/StaticFileContext.cs

```diff
@@ -356,7 +357,7 @@ public Task SendStatusAsync(int statusCode)
             Stream readStream = _fileInfo.CreateReadStream();
             try
             {
-                await StreamCopyOperation.CopyToAsync(readStream, _response.Body, _length, _context.RequestAborted);
+                await StreamCopyOperation.CopyToAsync(readStream, _response.Body, _length, StreamCopyBufferSize, _context.RequestAborted);
```

Comment:
Could you add your comment here that says this is required since `readStream` isn't going to be buffering?